app/models/collection.rb

```diff
@@ -443,4 +443,10 @@ def set_link_help
     end
   end
 
+  def user_help
```

Comment:
Wouldn't it be a bit simpler to override the `help` accessor to return `self[:help] || self.owner.help`, so we wouldn't need to do checks when we display?

Reply:
Note that if we do this, we will need to add a mutator override that makes sure that `collection[:help]` is set to `nil` if the user tries to set it to a value identical to `collection.owner.help`.  Otherwise we might end up with this scenario:
* Owner sets the help text on their profile to `AAAA`
* Owner edits a collection and modifies the description (or some value other than the help text)
* _`collection.help` is accidentally set to `AAAA`_
* Owner sets the help text on their profile to `BBBB`
* User views the collection help, expecting to see `BBBB` but instead sees `AAAA`

Reply:
@benwbrum In the view it was `@collection.help` previously which is pointing to help attribute of `@collection` instance variable. i.e we do not have any help accessor. If I create a new class method in collection model named as help to `return  self[:help] || self.owner.help` which i tried but it makes things unnecessarily cumbersome which I think is not a good approach.

Reply:
Fair enough.